lib/passkit/factory.rb

```diff
@@ -2,8 +2,8 @@ module Passkit
   class Factory
     class << self
       # generator is an optional ActiveRecord object, the application data for the pass
-      def create_pass(pass_class, generator = nil)
-        pass = Passkit::Pass.create!(klass: pass_class, generator: generator)
+      def create_pass(pass_class, generator: nil, pass_attributes: {})
```

Comment:
I’d keep the fork as close as possible to upstream.
```suggestion
      def create_pass(pass_class, generator = nil, pass_attributes = {})
```